botocore/loaders.py

```diff
@@ -205,9 +207,11 @@ class Loader(object):
     # For convenience we automatically add ~/.aws/models to the data path.
     CUSTOMER_DATA_PATH = os.path.join(os.path.expanduser('~'),
                                       '.aws', 'models')
+    BUILTIN_TOOL_EXTRAS = ['sdk', 'botocore']
```

Comment:
I agree on removing the `tool` part.  It's really up to the higher level layers to decide the semantics of the file.  From the loaders perspective they're just files that get deep merged on top of other files.  I'd prefer something like extras or extras_types as well.  

I'd also prefer to just have a single built in extra type (either botocore or sdk) for now unless we really need to have two.